api-report/tinylicious-client.api.md

```diff
@@ -28,9 +29,9 @@ export class TinyliciousAudience extends ServiceAudience<TinyliciousMember> impl
 
 // @public
 class TinyliciousClient {
-    constructor(connectionConfig?: TinyliciousConnectionConfig, logger?: ITelemetryBaseLogger | undefined);
+    constructor(props?: TinyliciousClientProps | undefined);
     createContainer(containerSchema: ContainerSchema): Promise<{
-        container: IFluidContainer;
+        container: FluidContainer;
```

Comment:
This was recently changed to the interface -- is it intentional to change it back?

Reply:
@alpaix , let's revert this for now back to `IFluidContainer`

Reply:
Rather a rebase artifact. Will revert.